piker/testing/__init__.py

```diff
@@ -1 +1 @@
-TEST_CONFIG_DIR_PATH = '_testing' 
+TEST_CONFIG_DIR_PATH = '/_testing' 
```

Comment:
hmm this isn't going to be cross-OS fyi; not that it's needed right now obviously 😂 

Can you instead lever `pathlib` APIs to do this correctly?